rtcrtpreceiver.go

```diff
@@ -49,6 +49,7 @@ func (r *RTCRtpReceiver) Receive(parameters RTCRtpReceiveParameters) chan bool {
 		}()
 
 		srtpSession, err := r.transport.getSRTPSession()
+		defer srtpSession.Close()
```

Comment:
nice!

Mind moving this after the `err` handling, we don't need to close if this fails.